modules/simple-payments/simple-payments.php

```diff
@@ -286,6 +287,134 @@ function allow_sync_post_meta( $post_meta ) {
+	 * to that meta key.
+	 *
+	 * @link https://developer.wordpress.org/rest-api/extending-the-rest-api/modifying-responses/
+	 */
+	public function register_meta_fields_in_rest_api() {
+		register_meta( 'post', 'spay_price', array(
+			'description'       => esc_html__( 'Simple payments; price.', 'jetpack' ),
+			'object_subtype'    => self::$post_type_product,
+			'sanitize_callback' => array( $this, 'sanitize_price' ),
+			'show_in_rest'      => true,
+			'single'            => true,
+			'type'              => 'number',
+		) );
+
+		register_meta( 'post', 'spay_currency', array(
+			'description'       => esc_html__( 'Simple payments; currency code.', 'jetpack' ),
+			'object_subtype'    => self::$post_type_product,
+			'sanitize_callback' => array( $this, 'sanitize_currency' ),
+			'show_in_rest'      => true,
+			'single'            => true,
+			'type'              => 'string',
+		) );
+
+		register_meta( 'post', 'spay_cta', array(
+			'description'       => esc_html__( 'Simple payments; text with "Buy" or other CTA', 'jetpack' ),
+			'object_subtype'    => self::$post_type_product,
+			'sanitize_callback' => 'sanitize_text_field',
+			'show_in_rest'      => true,
+			'single'            => true,
+			'type'              => 'string',
+		) );
+
+		register_meta( 'post', 'spay_multiple', array(
+			'description'       => esc_html__( 'Simple payments; allow multiple items', 'jetpack' ),
+			'object_subtype'    => self::$post_type_product,
+			'sanitize_callback' => 'absint',
+			'show_in_rest'      => true,
+			'single'            => true,
+			'type'              => 'integer',
+		) );
+
+		register_meta( 'post', 'spay_email', array(
+			'description'       => esc_html__( 'Simple payments button; paypal email.', 'jetpack' ),
+			'sanitize_callback' => 'sanitize_email',
+			'show_in_rest'      => true,
+			'single'            => true,
+			'type'              => 'string',
+		) );
+
+		register_meta( 'post', 'spay_formatted_price', array(
+			'description'       => esc_html__( 'Simple payments; formatted price.', 'jetpack' ),
+			'object_subtype'    => self::$post_type_product,
+			'sanitize_callback' => 'sanitize_text_field',
+			'show_in_rest'      => true,
+			'single'            => true,
+			'type'              => 'string',
+		) );
+
+		register_meta( 'post', 'spay_status', array(
+			'description'       => esc_html__( 'Simple payments; status.', 'jetpack' ),
+			'object_subtype'    => self::$post_type_product,
+			'sanitize_callback' => 'sanitize_text_field',
+			'show_in_rest'      => true,
+			'single'            => true,
+			'type'              => 'string',
+		) );
+	}
+
+	/**
+	 * Sanitize three-character ISO-4217 Simple payments currency
+	 *
+	 * @link https://developer.paypal.com/docs/integration/direct/rest/currency-codes/
+	 *
+	 * List has to be in sync with list at the client side:
+	 * @link https://github.com/Automattic/wp-calypso/blob/6d02ffe73cc073dea7270a22dc30881bff17d8fb/client/lib/simple-payments/constants.js
```

Comment:
Probably worth noting that both should align with PayPal (has a comment there):

https://developer.paypal.com/docs/integration/direct/rest/currency-codes/


https://github.com/Automattic/wp-calypso/blob/6d02ffe73cc073dea7270a22dc30881bff17d8fb/client/lib/simple-payments/constants.js#L4